plugins/verification/src/main/java/org/qbicc/plugin/verification/ClassLoadingBasicBlockBuilder.java

```diff
@@ -160,6 +161,16 @@ private BasicBlock noClassDefFound(TypeDescriptor desc) {
         return throw_(ncdfe);
     }
 
+    private BasicBlock verifyError(TypeDescriptor desc) {
+        ctxt.warning(getLocation(), "Reference to %s always produces VerifyError", desc);
+        Info info = Info.get(ctxt);
+        ClassTypeDescriptor ncdfeClass = info.ncdfeClass;
+        // todo: add class name to exception string
+        Value ncdfe = new_(ncdfeClass);
+        call(constructorOf(ncdfe, ncdfeClass, MethodDescriptor.VOID_METHOD_DESCRIPTOR), List.of());
+        return throw_(ncdfe);
```

Comment:
Did you intend to use `veClass` here instead?

Reply:
yep.  sloppy cut & paste.  fixed